client/src/components/Sidebar/Chat.js

```diff
@@ -23,11 +23,31 @@ const Chat = (props) => {
   const classes = useStyles();
   const { conversation } = props;
   const { otherUser } = conversation;
+  const { messages } = conversation;
+  // Will disable badge showing unread messages when we are currently in the chat window
+  const { activeConversation } = props
```

Comment:
You can group these destructurings:

```suggestion
  const { conversation, activeConversation } = props;
  const { otherUser, messages } = conversation;
```

Reply:
destructuring was done in groups